app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -73,5 +77,21 @@ def textual_compliance_history
       end
       h
     end
+
+    def failed_rules_summary
+      @failed_rules_summary ||= @record.openscap_failed_rules_summary
+    end
+
+    def textual_low
```

Comment:
Since these methods are all in the same module, textual_low by itself is meaningless (and thus hard to read).  Prefer more explicit `textual_openscap_failed_rules_low`, etc.